packages/ember-metal/lib/mixin.js

```diff
@@ -252,9 +252,9 @@ function applyMergedProperties(obj, key, value, values) {
   }
 
   return newBase;
-}
+};
 
-function addNormalizedProperty(base, key, value, meta, descs, values, concats, mergings) {
+var addNormalizedProperty = (base, key, value, meta, descs, values, concats, mergings) => {
```

Comment:
On thing I'd consider from a style-guide sense, is to keep the named function form if possible. It's much cleaner, and easy to see the functions.

Arrow functions should be used when functions are passed to functions, like:

```js
items.forEach(item => parent.remove(item));
```

Arrow functions could be a way into sloppy programming where the context is passed around everywhere. We want to only preserve the context when necessary.